hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/DropHoodieTableCommand.scala

```diff
@@ -45,10 +45,7 @@ case class DropHoodieTableCommand(
     if (!sparkSession.catalog.tableExists(tableIdentifier.unquotedString)) {
       sparkSession.catalog.refreshTable(tableIdentifier.unquotedString)
     }
-    val qualifiedTableName = QualifiedTableName(
-      tableIdentifier.database.getOrElse(sparkSession.sessionState.catalog.getCurrentDatabase),
-      tableIdentifier.table)
-    sparkSession.sessionState.catalog.invalidateCachedTable(qualifiedTableName)
+    sparkSession.sessionState.catalog.invalidateCachedTable(tableIdentifier)
```

Comment:
Does this cause backwards compatibility issue on Spark 3.5.1?  Does Spark 3.5.3 changes this API from Spark 3.5.1 to 3.5.3?

Reply:
It looks like this API is added in Spark 3.2 so we can improve the code by using it instead (master branch supports Spark 3.3 and above).